packages/analytics/amplify_analytics_pinpoint/lib/src/flutter_endpoint_info_store_manager.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:amplify_analytics_pinpoint/src/legacy_native_data_provider/flutter_legacy_native_data_provider.dart';
+import 'package:amplify_analytics_pinpoint_dart/amplify_analytics_pinpoint_dart.dart';
+import 'package:amplify_secure_storage/amplify_secure_storage.dart';
+
+/// {@template amplify_analytics_pinpoint_dart.flutter_endpoint_store_manager}
+/// Manages retrieval, storage, and management of Pinpoint Endpoint id and fields
+/// Auto injects Flutter DI
+/// {@endtemplate}
+class FlutterEndpointInfoStoreManager extends EndpointInfoStoreManager {
+  /// {@macro amplify_analytics_pinpoint_dart.flutter_endpoint_store_manager}
+  FlutterEndpointInfoStoreManager()
+      : super(
+          store: AmplifySecureStorage(
```

Comment:
This class will be exported to Cognito Auth to allow Auth to manage creation and retrieval of endpoint id.  

I reduce the DI here so that Auth can simply use this class without writing any additional config logic.  Doing so reduces risk of decoupled endpoint logic between Auth and Analytics.